runtime/vm/resolvesupport.cpp

```diff
@@ -451,6 +451,26 @@ resolveStaticMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA cp
 	} else {
 		cpClass = J9_CLASS_FROM_CP(ramCP);
 		lookupOptions |= J9_LOOK_CLCONSTRAINTS;
+
+		if (cpClass != NULL && cpClass->romClass != NULL) {
+			if (isResolvedClassAnInterface) {
+				if ((J9CPTYPE_INTERFACE_STATIC_METHOD != J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex)) &&
+					(J9CPTYPE_INTERFACE_INSTANCE_METHOD != J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex)) &&
+					(J9CPTYPE_INTERFACE_METHOD != J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex))) {
```

Comment:
formatting - put the closing `) {` on their own line indented to the same level as the `if`

Reply:
Pull `J9_CP_TYPE(J9ROMCLASS_CPSHAPEDESCRIPTION(cpClass->romClass), cpIndex)` into a local variable.